x-pack/plugins/maps/public/index.ts

```diff
@@ -17,7 +17,8 @@ export const plugin: PluginInitializer<MapsPluginSetup, MapsPluginStart> = (
   return new MapsPlugin(initContext);
 };
 
-export { MAP_SAVED_OBJECT_TYPE } from '../common/constants';
```

Comment:
Instead of exporting APP_ID in maps/public/index, an ML just import from `maps/common`. You will have to add `APP_ID` to `maps/common/index.ts`

Reply:
Good catch - moved to export from /common in 3bfae94c05b9d5ccd1dae877e83cf7a374e59b3e